apps/frontend/src/components/global/upload_tabs/DatabaseReader.vue

```diff
@@ -92,6 +94,20 @@ export default class DatabaseReader extends mixins(ServerMixin) {
     });
   }
 
+  async check_shared_evaluations(): Promise<void> {
+    const evaluationsToLoad: string = Vue.$cookies.get('loadEvaluation');
+    if(evaluationsToLoad){
+      const evaluationsToLoadArray: IEvaluation[] = await EvaluationModule.findEvaluationsByIds(evaluationsToLoad.split(','));
+      if (evaluationsToLoadArray.length !== 0) {
+        this.load_results(evaluationsToLoadArray);
+        Vue.$cookies.remove('loadEvaluation');
+      } else {
+        SnackbarModule.failure(`Heimdall was passed the following evaluations to open, but couldn't find any of them: ${evaluationsToLoad}`);
+        Vue.$cookies.remove('loadEvaluation');
+      }
+    }
+  }
```

Comment:
The DatabaseReader and the UploadNexus really don't feel like the place that this should be handled, this seems more like a vue router or a Results component item.